qa/objectstore/bluestore.yaml

```diff
@@ -20,3 +20,21 @@ overrides:
         osd failsafe full ratio: .95
 # this doesn't work with failures bc the log writes are not atomic across the two backends
 #        bluestore bluefs env mirror: true
+  ceph-deploy:
+    fs: xfs
+    bluestore: yes
```

Comment:
we probably need to add overrides to filestore-xfs.yaml in this same dir with filestore: true.. otherwise *all* tests willb e bluestore tests, since it's the new default, right?